packages/flutter_test/lib/src/binding.dart

```diff
@@ -606,7 +605,7 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
     ui.window.onDrawFrame = null;
   }
 
-  FakeAsync _fakeAsync;
+  FakeAsync _currentFakeAsync; // set from runTest to postTest
```

Comment:
This is slightly ambiguous as to being inclusive vs exclusive - maybe instead:

```dart
// set in [runTest]; cleared in [postTest]
```

Reply:
Yeah, I wasn't sure how to phrase these (there are several). I will update this in the next PR.